spec.emu

```diff
@@ -318,7 +318,13 @@ contributors: Ujjwal Sharma, Younies Mahmoud
               1. For each Record { [[Type]], [[Value]], [[Unit]] } _part_ in _parts_, do
                 1. Set _concat_ to the string-concatenation of _concat_ and _part_.[[Value]].
               1. Append the new Record { [[Type]]: _unit_, [[Value]]: _concat_ } to the end of _result_.
-        1. Let _lf_ be ! Construct(%ListFormat%, &laquo; _durationFormat_.[[Locale]] &raquo;).
+        1. Let _lfOpts_ be ! OrdinaryObjectCreate(*null*).
+        1. Perform ! CreateDataPropertyOrThrow(_lfOpts_, *"type"*, *"unit"*).
```

Comment:
This PR made me realize why setting the `"type"` is important, however, I think we need to set a different type based on the style.